traffic_portal/build/etc/init.d/traffic_portal

```diff
@@ -46,14 +62,14 @@ start() {
       -l $LOGFILE \
       --restart-delay $RESTART_TIME \
       --time \
-      --name "$NAME"
+      --name "$NAME" 2>&1 > /dev/null
     RETVAL=$?
 }
 
 stop() {
     if pm2 list | grep -q "$NAME"; then
         echo "Shutting down $NAME"
-        pm2 stop $APPLICATION_PATH
+        pm2 stop $APPLICATION_PATH 2>&1 > /dev/null
```

Comment:
`2>&1` needs to go after `> /dev/null` or it doesn't redirect `/dev/stderr` to `/dev/null`

Reply:
Will change, I was just copy & pasting what the script was doing previously.